### PR TITLE
Add tests for testing module

### LINKED ISSUE #146: Add tests for the testing subpackage
The simphony.testing subpackage does not have any tests.

### PR DESCRIPTION
**Pull request by itziakos**
This PR fixes #146

### CODE REVIEW

**Comment by itziakos** [comment]
@roigcarlo and @SGGgarcia, please have a look

File: simphony/testing/tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+        bond = Bond([None])
+
+        # when/then
+        with self.assertRaises(AssertionError):
+            compare_bonds(bond, reference, testcase=self)
+
+
+class TestCompareElements(unittest.TestCase):
+
+    def test_compare_bonds_equal(self):
```
Comment (roigcarlo) [inline simphony/testing/tests/test_utils.py]: compare_elements_equal

File: simphony/testing/tests/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+        # given
+        element = Element(
+            uid=None,
+            points=list(reversed(points)),
+            data=create_data_container())
+
+        # this should pass without problems
+        compare_elements(element, reference, testcase=self)
+
+    def test_compare_bonds_not_equal(self):
```
Comment (roigcarlo) [inline simphony/testing/tests/test_utils.py]: compare_elements_not_equal

**Comment by SGGgarcia** [comment]
Besides de typos, it's ok :+1:

**Comment by itziakos** [comment]
Done, will merge as soon as tests pass

**Pull request merged**